app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -409,6 +423,16 @@ public void onUnreadItemsChanged(UnreadItemsUpdateEvent event) {
         updateUi();
     }
 
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void favoritesChanged(FavoritesEvent event) {
+        updateUi();
+    }
+
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onQueueChanged(QueueEvent event) {
+        updateUi();
+    }
+
```

Comment:
This always refreshes the whole list now. When the list is huge, it could be sluggish or break animations. Can't this be handled with a `FeedItemEvent` that only updates a single item?

Reply:
Can you test if this is an actual problem? I actually just reused code from the buttons and the context menu, so it wouldn't be slower or sluggier then when your select a context menu option.
I think the list refresh is requiered because sometimes the database is changed and sometimes the item is removed from the list or just changes status.
I think it's enough of the loading progress bar is displayed if the list is empty in the beginning, afterwards it's just looks wired of the progress is flashing up in front of the items for a milisec...